Sources/SKSwiftPMWorkspace/SwiftPMWorkspace.swift

```diff
@@ -113,13 +113,7 @@ public final class SwiftPMWorkspace {
     swiftPMToolchain.extraSwiftCFlags = extraSwiftFlags
     swiftPMToolchain.extraCPPFlags = extraClangFlags
 
-
-    let buildPath: AbsolutePath
-    if let absoluteBuildPath = try? AbsolutePath(validating: buildSetup.path) {
-      buildPath = absoluteBuildPath
-    } else {
-      buildPath = packageRoot.appending(component: buildSetup.path)
-    }
+    let buildPath: AbsolutePath = buildSetup.path ?? packageRoot.appending(component: ".build")
```

Comment:
SwiftPM can use its default by any chance?

Reply:
It's not currently exposed in a way we can use it.  There is `Workspace.create(forRootPackage:manifestLoader:)`, which will use the default path, but it doesn't let you customize it or use an in-memory filesystem (which we use in tests).